jodie.py

```diff
@@ -239,6 +231,14 @@
                    
                     # REINITIALIZE
                     if is_first_epoch:
+                        cached_tbatches[timestamp] = lib.current_tbatches_user
+                        cached_tbatches[timestamp] = lib.current_tbatches_item
```

Comment:
Hello. current_tbatches_item seems to mask current_tbaches_user since they are saved in the same place. Is this intentional? Thank you!!

Reply:
@SungMinCho thanks for noticing. I have fixed and pushed an update. Please notify us if you find any other bugs. Thanks!

Reply:
👍 Thx for the catching the bug and get it fixed, sorry for the typo.